src/core/tests/type_prop/reverse_sequence.cpp

```diff
@@ -78,7 +78,7 @@ TEST(type_prop, reverse_sequence_invalid_seq_lengths_et) {
 }
 
 TEST(type_prop, reverse_sequence_invalid_data_rank) {
-    std::vector<PartialShape> invalid_pshapes = {{}, {4}};
+    std::vector<PartialShape> invalid_pshapes = {{4}};
```

Comment:
Do we really need this change?

Reply:
my bad, this is a debug change, will revert it